packages/cf-style-container/src/index.js

```diff
@@ -49,9 +56,40 @@ const createComponentStyles = (styleFunctions, component) =>
 
 export {
   createComponent,
+  mergeThemes,
   applyTheme,
   ThemeProvider,
   connect,
   combineRules,
-  createComponentStyles
+  createComponentStyles,
+  capitalize
 };
+
+// Loops the key-value pairs of a props object, and apply a filter function to
+// every pair. The returned value from the filter function will be reduced back
+// to a single object.
+export const filterProps = (props, filter) =>
+  Object.keys(props)
+    .map(key => [key, props[key]])
+    .reduce((accum, [key, value]) => filter(key, value, accum), {});
+
+// Filter out all the keys with null or undefined values
+export const filterNone = props =>
+  filterProps(props, (key, value, accum) => {
+    if (value == null) return accum;
+    accum[key] = value;
+    return accum;
+  });
+
+// Filter out all the style attribute from the props
+export const filterStyle = props =>
+  filterProps(props, (key, value, accum) => {
+    if (key === 'style') return accum;
+    accum[key] = value;
+    return accum;
+  });
+
+export const mapChildren = (children, func) =>
```

Comment:
What about renaming this mapChildrenWithIndex to indicate why it differs from React.Children.map?